build.gradle

```diff
@@ -24,8 +24,8 @@ allprojects {
             name "vsts-maven-adal-android"
             url "https://identitydivision.pkgs.visualstudio.com/_packaging/AndroidADAL/maven/v1"
             credentials {
-                username project.vstsUsername
-                password project.vstsPassword
+                username project.findProperty("vstsUsername")
```

Comment:
Oops. Just realized that this is now using a different config from CI (top of this file) -- I'll send out a PR to make everything use the same value